src/main.cpp

```diff
@@ -336,24 +344,26 @@ static void list_game_ids()
 		std::cout << gameid <<std::endl;
 }
 
-static void list_worlds()
+static void list_worlds(bool print_name, bool print_path)
 {
 	std::cout << _("Available worlds:") << std::endl;
 	std::vector<WorldSpec> worldspecs = getAvailableWorlds();
-	print_worldspecs(worldspecs, std::cout);
+	print_worldspecs(worldspecs, std::cout, print_name, print_path);
 }
 
 static void print_worldspecs(const std::vector<WorldSpec> &worldspecs,
-							 std::ostream &os)
+	std::ostream &os, bool print_name, bool print_path)
 {
 	for (const WorldSpec &worldspec : worldspecs) {
 		std::string name = worldspec.name;
 		std::string path = worldspec.path;
-		if (name.find(' ') != std::string::npos)
-			name = std::string("'").append(name).append("'");
-		path = std::string("'").append(path).append("'");
-		name = padStringRight(name, 14);
-		os << "  " << name << " " << path << std::endl;
+		if (print_name && print_path) {
+			os << "\t" << name << "\t\t" << path << std::endl;
+		} else if (print_name) {
+			os << "  " << name << std::endl;
+		} else if (print_path) {
+			os << "  " << path << std::endl;
```

Comment:
`"\t" << path` and above `" \t" << name` should be used for consistency.
Either that or `" " << name` for the first case. Both is fine by me.